recipes/huffman/all/CMakeLists.txt

```
@@ -0,0 +1,42 @@
+﻿cmake_minimum_required (VERSION 3.4)
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE STRING "Minimum OS X deployment version")
+project (huffman)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MSVC)
+    add_definitions (-D_WIN32_WINNT=0x0601)
+    add_definitions (-D_SCL_SECURE_NO_WARNINGS=1)
+    add_definitions (-D_CRT_SECURE_NO_WARNINGS=1)
+
+    set(CMAKE_VS_INCLUDE_INSTALL_TO_DEFAULT_BUILD 1)
+
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc /utf-8")
```

`CMAKE_VS_INCLUDE_INSTALL_TO_DEFAULT_BUILD` is only potential useful when developing the library, not for creating packages.
```suggestion
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc /utf-8")
```